Gemfile

```diff
@@ -18,7 +18,7 @@ gem "administrate", "~> 0.14" # A Rails engine that helps you put together a sup
 gem "ahoy_email", "~> 1.1" # Email analytics for Rails
 gem "ahoy_matey", "~> 3.0" # Tracking analytics for Rails
 gem "ancestry", "~> 3.1" # Ancestry allows the records of a ActiveRecord model to be organized in a tree structure
-gem "autoprefixer-rails", "~> 9.8" # Parse CSS and add vendor prefixes to CSS rules using values from the Can I Use website
+gem "aws-sdk-lambda", "~> 1.48" # Official AWS Ruby gem for AWS Lambda
```

Comment:
I think there was a merge conflict somewhere, this gem has been removed by https://github.com/forem/forem/pull/9978/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L22

Reply:
Yeah, just sorting this out.